src/core/memblock.c

```diff
@@ -0,0 +1,34 @@
+#include "memblock.h"
+
+#include <stdlib.h>
+
+int core_memblock_init(memblock *block, uint32_t initial_size)
```

Comment:
`memory_block` instead of `memblock` here.

Reply:
I can change it, but I would like to point out Bianca’s final ruling was that using abbreviations was ok as long as they’re consistent with what is used elsewhere in the code. So dst was considered ok but not dest.

Reply:
Yeah but "mem" isn't really used elsewhere.